css/deeplinking.scss

```diff
@@ -0,0 +1,73 @@
+.deeplinking {
+    background-color: #fff;
+    width: 100%;
+    height: 100%;
+    .header {
+        width: 100%;
+        height: 70px;
+        background-color: #f1f2f5;
+        .logo {
+            margin-top: 15px;
+            margin-left: 50px;
+            height: 40px;
+            float: left;
```

Comment:
Why do you need a float? You can't use inline-block?

Reply:
Removed!